lib/kubernetes-deploy/runner.rb

```diff
@@ -19,11 +19,11 @@ def self.with_friendly_errors
       exit 1
     end
 
-    def initialize(namespace:, environment:, current_sha:, template_folder: nil, context:)
+    def initialize(namespace:, environment:, current_sha:, context:, template_dir: nil)
       @namespace = namespace
       @context = context
       @current_sha = current_sha
-      @template_path = File.expand_path('./' + (template_folder || "config/deploy/#{environment}"))
+      @template_dir = template_dir || File.expand_path("./config/deploy/#{environment}")
```

Comment:
I see. We should totally make it use a relative argument as you suggested.
I'll update the PR and shipit.